ui/pages/settings/info-tab/info-tab.component.js

```diff
@@ -28,6 +28,10 @@ import {
 } from '../../../../shared/lib/ui-utils';
 
 export default class InfoTab extends PureComponent {
+  static propTypes = {
+    remoteFeatureFlags: PropTypes.object,
```

Comment:
This should be marked as required; the component will crash on mount if it's missing

```suggestion
    remoteFeatureFlags: PropTypes.object.isRequired,
```

Reply:
Added 545943e3a603e160da198d220ddd14c21213b347